test/helpers/jobs.ts

```diff
@@ -73,6 +73,20 @@ export function areJobLinksEqual(jobLinks: JobLink[], serializedJobLinks: JobLin
   return _.isEqual(cleanedLinksSet, serializedLinksSet);
 }
 
+/**
+ * Compare link titles from a saved Job to those of the STAC catalog output
+ * @param jobLinks - links from a saved job
+ * @param stacLinks - links from STAC catalog
+ * @returns true if the links are the same, false otherwise
+ */
+export function areStacJobLinksEqual(jobLinks: JobLink[], stacLinks: JobLink[]): boolean {
+  const cleanedStacLinks = stacLinks.map((link) => _.pick(link, ['title']));
```

Comment:
Shouldn't we compare all the STAC link fields like temporal and bbox fields as well?

Reply:
From my understanding, for the STAC _catalog_, temporal and bbox fields aren't included in the output links. HREF also won't ever be equal to the original data links HREF. And the rel for STAC links in a catalog are 'item', so they will never be equal to 'data'. Title was the only comparable property I could find.

e.g. STAC catalog:

```
{
  "id": "faf56600-98e4-45d6-871e-0a26ae0a3651",
  "stac_version": "0.9.0",
  "title": "Harmony output for faf56600-98e4-45d6-871e-0a26ae0a3651",
  "description": "Harmony output for http://localhost:3000/C1233800302-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?granuleId=G1233800343-EEDTEST&forceAsync=true",
  "links": [
    {
      "href": ".",
      "rel": "root",
      "title": "root"
    },
    {
      "href": "./0",
      "rel": "item",
      "title": "001_00_7f00ff_global.nc.tif"
    },
    {
      "title": "The current page",
      "href": "http://localhost:3000/stac/faf56600-98e4-45d6-871e-0a26ae0a3651?limit=500&page=1",
      "rel": "self",
      "type": "application/json"
    }
  ]
}
```